.github/workflows/codeql.yml

```diff
@@ -48,3 +61,28 @@ jobs:
         uses: github/codeql-action/analyze@6a28655e3dcb49cb0840ea372fd6d17733edd8a4 # v2.21.8
         with:
           category: "/language:${{matrix.language}}"
+        outputs:
+          sarif-output: /tmp/codeql-results.sarif
```

Comment:
have you tried this? 

I am wondering if what the action documents is actually an output of the action, that can be then accessed with something like ` ${{ steps.analyze.outputs.sarif-output}}`

https://github.com/github/codeql-action/blob/4ebadbc7468649cf79b138f45e20d999351f6ed0/analyze/action.yml#L81

Reply:
Added, the only problem is that it does not include the filename. I've added the changes so it uses the default one along with the absolute path take from the action.